lib/_emerge/DepPriorityNormalRange.py

```diff
@@ -37,7 +37,10 @@ def _ignore_runtime_post(cls, priority):
     def _ignore_runtime(cls, priority):
         if priority.__class__ is not DepPriority:
             return False
-        return bool(priority.optional or not priority.buildtime)
+        return bool(
+            not priority.runtime_slot_op
+            and (priority.optional or not priority.buildtime)
+        )
```

Comment:
This sort of perverts the meaning of `_ignore_runtime`, since it is no longer possible to ignore an unsatisfied runtime_slot_op dependency. The perversion is good in the sense that it gives a strict merge order guarantee for unsatisfied runtime_slot_op dependencies.

Reply:
If we ever added support for an "optional" runtime_slot_op then we would need to adjust this code to account for that, but it's fine for now since only buildtime dependencies can currently be optional. Maybe a --onlydeps runtime_slot_op could be considered optional.